src/Kconfig.storage

```diff
@@ -34,26 +34,29 @@ config THINGSET_STORAGE_LOAD_ATTEMPTS
 	  with bad CRCs due to noise in the communication. This Kconfig defines how often reading
 	  from the storage should be retried.
 
-config THINGSET_STORAGE_CHANGE
-	bool "Store data upon change to ThingSet stored items"
+config THINGSET_STORAGE_SAVE_UPDATES
+	bool "Store data upon updates to ThingSet stored items"
 	default y
 	help
-	  If enabled, a callback is invoked once a data object which is part of the TS_SUBSET_NVM is
-	  changed.
+	  If enabled, data objects which are part of the TS_SUBSET_NVM subset are immediately stored
+	  in NVM after an update of one of these data objects via ThingSet.
 
-config THINGSET_STORAGE_REGULAR
+config THINGSET_STORAGE_SAVE_REGULARLY
```

Comment:
My only comment on this is is that "regularly" is often used to mean "often" rather than "at even intervals" (see e.g. https://dictionary.cambridge.org/us/dictionary/english/regularly). This functionality is generally referred to as autosave, thus:

`THINGSET_STORAGE_ENABLE_AUTOSAVE`

...and then...

Reply:
Good remark. Fixed.